packages/api/test/e2e/rx-tx.spec.ts

```diff
@@ -26,14 +28,14 @@ describe.skip('e2e transactions', () => {
       .accountNonce(keyring.alice.address())
       .pipe(
         first(),
-        switchMap((nonce) =>
+        switchMap((nonce: any) =>
```

Comment:
`: Index`

Reply:
Same `Codec`  error ashere https://github.com/polkadot-js/api/pull/936#discussion_r287831252 (probably with the same solution)